index.js

```diff
@@ -156,15 +156,16 @@ module.exports = {
     'id-length': 0,
     'id-match': 0,
     'indent': [
-      0,
-      2
+      2,
+      2,
+      { 'SwitchCase': 1 }
     ],
     'jsx-quotes': 0,
     'key-spacing': 0,
     'linebreak-style': 0,
     'lines-around-comment': 0,
     'max-depth': 0,
-    'max-len': 0,
+    'max-len': [2, { 'code': 100 }],
```

Comment:
👍 

https://eslint.org/docs/rules/max-len#enforce-a-maximum-line-length-max-len

Might want to allow trailing comments as well

Reply:
This is the rule used by the airbnb config that omits certain things from max-len: https://github.com/airbnb/javascript/blob/17e0454672f784772c5f5ab052523a93229d9adf/packages/eslint-config-airbnb-base/rules/style.js#L195. Since the rule in this PR specifies only code, I'm wondering if these are just two ways of doing the same thing or if there will be slightly different results.